### PR TITLE
MINOR: Fix wrong polecap color for here-webtile example

### PR DESCRIPTION
**Pull request by ninok**

### CODE REVIEW

File: @here/harp-examples/src/datasource_here-webtile.ts [hunk 1]
```diff
@@ -7,8 +7,46 @@ import { MapControls, MapControlsUI } from "@here/harp-map-controls";
 import { CopyrightElementHandler, MapView } from "@here/harp-mapview";
 import { HereWebTileDataSource, WebTileDataSource } from "@here/harp-webtile-datasource";
 
+import { Theme } from "@here/harp-datasource-protocol";
+import { ProjectionType } from "@here/harp-geoutils";
 import { apikey } from "../config";
 
+const FLAT_THEME: Theme = {
+    clearColor: "#f8fbfd"
```
Comment (nzjony) [inline @here/harp-examples/src/datasource_here-webtile.ts:15]: don't you need an `extends` ?

Comment (ninok) [reply @here/harp-examples/src/datasource_here-webtile.ts:15]: no b/c the WebTile source needs no styling

**Comment by ninok** [comment]
@harpgl-bot retest this please

File: @here/harp-examples/src/datasource_here-webtile.ts (hunk 1 above)
Comment (nzjony) [reply @here/harp-examples/src/datasource_here-webtile.ts:15]: Ok, cool.

**Pull request merged**